polkadot-parachain/src/command.rs

```diff
@@ -142,8 +146,17 @@ fn load_spec(id: &str) -> std::result::Result<Box<dyn ChainSpec>, String> {
 		)?),
 		// -- Fallback (generic chainspec)
 		"" => Box::new(chain_spec::get_chain_spec()),
+		"penpal-kusama" => Box::new(chain_spec::penpal::get_penpal_chain_spec(
+			para_id.expect("Must specify parachain id"),
+			"kusama-local",
+		)),
+		"penpal-polkadot" => Box::new(chain_spec::penpal::get_penpal_chain_spec(
+			para_id.expect("Must specify parachain id"),
+			"polkadot-local",
+		)),
 		// -- Loading a specific spec from disk
 		path => {
+			println!("could not find a path {}", path);
```

Comment:
```suggestion
```

Reply:
any stdout println messes up generation of raw json because we pipe output to a file